src/main/java/com/example/universe/simulator/entityservice/services/GalaxyService.java

```diff
@@ -18,6 +22,7 @@
 
 @Service
 @Transactional(readOnly = true)
+@CacheConfig(cacheNames = "galaxy")
```

Comment:
As you're using this string in `GalaxyIntegrationTest` too, it would be cleaner to externalize it as a spring property.

Action points:

1. Add a new field (with `@NotBlank` validation and javadoc comment) in `CachingProperties` in common. You can call it `galaxyCacheName` (same as ur constant name in `GalaxyIntegrationTest`.
3. add `@Validated` to `CachingProperties` for validation to work
4. add the new property in `application.properties` in the caching section
5. Change `@CacheConfig(cacheNames = "galaxy")` in `GalaxyService` to use the new property. Check the `@Scheduled` annotation in SpaceEntityStatisticsJob` if unclear.
6. Replace the `GALAXY_CACHE_NAME` constant in `GalaxyIntegrationTest` with a spring property (`@Value`)

Reply:
As long as @CacheConfig(cacheNames = ...) does not accept spring properties, above solution can't be done.
Created new constant GALAXY_CACHE_NAME  in GalaxyService instead.